sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVector.java

```diff
@@ -387,6 +449,11 @@ private void throwUnsupportedException(int requiredCapacity, Throwable cause) {
   public abstract byte getByte(int rowId);
 
   /**
+   * Gets values from [rowId, rowId + count) to [dst, dst + count)
```

Comment:
I see. Since the latest code stopped reusing the primitive type array, we made the API `public abstract byte[] getBytes(int rowId, int count);` that returns a primitive type array.